src/en/community/events/2022/ceph-virtual/index.md

```diff
@@ -277,9 +277,21 @@ Cybozu has been running and testing their Rook/Ceph clusters for two years. Duri
    </td>
    <td><strong>DATA SECURITY AND STORAGE HARDENING IN ROOK AND CEPH</strong>
 <p>
-1. The NIC SR-IOV is used. Each OSD uses an exclusive VF NIC. 2. The DPDK interrupt mode is added. 3. The single-CPU core and multiple NIC queues are implemented to improve performance. 4. The admin socket command is added to obtain the NIC status, collect statistics, and locate faults. 5. Adjust the CEPH throttling parameters, TCP, and DPDK packet sending and receiving buffer sizes to prevent packet loss and retransmission. 6. The Crimson message component uses the Seastar DPDK.
+This session shares an overview of troubleshooting a rook-ceph cluster, discussing some troubleshooting scenarios. This is followed by an introduction and an overview of kubectl-rook-ceph krew plugin, and how it makes managing containers easier from a troubleshooting perspective.
+We’ll also discuss future issues we’re planning to solve with it here, followed by a short future roadmap of the rook project.
+Later, we look forward to discussing and gathering feedback from the users about common and challenging problems they face while troubleshooting the clusters.
    </td>
    <td>Federico Lucifredi
```

Comment:
Is the data security and storage hardening talk still meant to be here? It doesn't look like the description/author match for that one

Reply:
looks like the description is accidentally from today's "Optimize Ceph messenger Performance" talk

Reply:
Whoops, thanks!